### DISCUSSION TITLE
LCD gauge option setting bar or mode

### DISCUSSION THREAD

**Discussion by PetersPlace** [grafana/grafana#38275] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
The LCD gauge works great but nice would be the option to set bar or dot mode.

**Why is this needed**:
This is nice if you want to monitor al value  between lets say 40 to 60 %, we set the tresholds green color between  40 and 60
Below 40 and above 60 is red color
With bar the 0-40 is always red colored, so i would be nice if only the green dots wil light up is the signal is between 40 and 60

![lcd-gauge](https://user-images.githubusercontent.com/35110891/117111623-3027bc00-ad88-11eb-9514-840be2079771.jpg)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
